src/main/java/seedu/address/model/person/Period.java

```diff
@@ -283,6 +270,24 @@ public static boolean isValidPeriodString(String toTest) {
         return true;
     }
 
+    /**
+     * Returns the first date of the period.
+     *
+     * @return LocalDate representing the first date of the period.
+     */
+    public LocalDate getStartDate() {
+        return startDate;
+    }
+
+    /**
+     * Returns a Period that represents a week that starts from the LocalDate provided.
+     *
+     * @param firstDate first date of the week.
+     */
+    public static Period oneWeekFrom(LocalDate firstDate) {
```

Comment:
This does not obtain from monday to sunday exact as specified? Why not use the ParserUtil::getWeekPeriodFromDate?

Reply:
I changed it from [Monday to Sunday] to [Today to next week] It doesn't make sense to display the next Monday behind today if its Tuesday now isn't it??

Reply:
As in currently, all the shift commands are doing it for the current week, as in monday to sunday. If we change it to display the next seven days including current, shouldn't we change the others as well?

Reply:
Should probably modify ParserUtil::getDateArrayOfWeek instead